websites/B/BigRigFM/presence.ts

```diff
@@ -0,0 +1,36 @@
+const presence = new Presence({
+		clientId: "985821439829622825",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "brfmlogo",
+		startTimestamp: browsingTimestamp,
+	};
```

Comment:
In regards to: https://github.com/PreMiD/Presences/pull/6323/files#r896137069 

```suggestion
	},
	spotifyUrl = document
		.querySelector(".player-artist-text a")
		.getAttribute("href");
```